test/grpc/route_guide/BUILD.bazel

```diff
@@ -0,0 +1,82 @@
+# NOTE: instead of 'cc_grpc_library' from '@com_github_grpc_grpc'
+# could also use 'cpp_grpc_library' from 'rules_proto_grpc' (by first
+# bringing in 'rules_proto_grpc' in WORKSPACE.bazel).
+load("@com_github_grpc_grpc//bazel:cc_grpc_library.bzl", "cc_grpc_library")
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_proto_library", "cc_test")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//bazel:cc_eventuals_library.bzl", "cc_eventuals_library")
+
+proto_library(
+    name = "route-guide",
+    srcs = ["route_guide.proto"],
+)
+
+cc_proto_library(
+    name = "route-guide-proto",
+    deps = [":route-guide"],
+)
+
+cc_grpc_library(
+    name = "route-guide-grpc",
+    srcs = [":route-guide"],
+    grpc_only = True,
+    deps = [":route-guide-proto"],
+)
+
+cc_eventuals_library(
+    name = "route-guide-eventuals-generated",
+    srcs = ["route_guide.proto"],
+    deps = [":route-guide"],
+)
+
+cc_library(
+    name = "route-guide-eventuals",
+    srcs = ["route-guide-eventuals-generated"],
+    deps = [
+        ":route-guide-grpc",
+        "//eventuals/grpc",
+    ],
+)
+
+cc_library(
```

Comment:
Can we instead just use https://github.com/grpc/grpc/blob/master/examples/cpp/route_guide/BUILD#L18?